Components/SwagImportExport/Transformers/ValuesTransformer.php

```diff
@@ -94,7 +94,7 @@ public function transform($type, $data)
                     foreach ($conversions as $variableName => $conversion) {
                         if (isset($record[$variableName]) && !empty($conversion)) {
                             $evalData = $this->evaluator->evaluate($conversion, $record);
-                            if ($evalData) {
+                            if ($evalData || $evaldata == "0") {
```

Comment:
please make sure that this is the right comparison you are using. https://3v4l.org/V8Qpv

it is always better to use `===` but then it is the question, what do you expect?